tests/services/inventory/crawling_test.py

```diff
@@ -290,7 +290,7 @@ def test_crawling_from_composite_root(self):
             'disk': {'resource': 3},
             'firewall': {'resource': 3},
             'folder': {'iam_policy': 2, 'resource': 2},
-            'forwardingrule': {'resource': 1},
```

Comment:
This one won't change as the resource is not included in the project getting tested with the composite root.

Reply:
Done